src/components/chip/chip.module.scss

```diff
@@ -1,55 +1,78 @@
+	--chip_label_padding-horizontal: var(--spc-1x);
+	--chip_icon_size: var(--icon-size_dense);
+
+	--chip_background-color: var(--transparent);
+	--chip_background-color_hovered: var(--surface_primary_emphasis-low);
+	--chip_background-color_pressed: var(--surface_primary_emphasis-high);
+	--chip_background-color_focused: var(--background_focus);
+	--chip_background-color_disabled: var(--transparent);
+
+	--chip_foreground-color: var(--foreground_emphasis-high);
+	--chip_foreground-color_disabled: var(--foreground_disabled);
+
+	--chip_outline_color: var(--primary_variant);
+	--chip_outline_color_disabled: var(--background_disabled);
+	--chip_outline_width: var(--border-width_m);
+	--chip_outline_offset: calc(0 - var(--chip_outline_width));
+
+	--chip_focus-outline_color: var(--focus-outline_primary);
+	--chip_focus-outline_width: var(--border-width_focus);
+}
+
+.chip {
+	height: var(--chip_min-size);
 	display: inline-flex;
 	align-items: center;
 	justify-content: center;
 	white-space: nowrap;
 
-	color: var(--foreground_emphasis-high);
+	color: var(--chip_foreground-color);
 	text-decoration: none;
 	outline: none !important;
 
-	@include transition(color background-color border-color box-shadow);
-
-	&:hover {
-		text-decoration: underline;
-	}
-
-	padding: var(--spc-1x) var(--spc-2x);
-
-	border: calc(var(--spc-1x) / 2) solid var(--primary_variant);
-	border-radius: 8px;
+	padding: var(--chip_padding-vertical) var(--chip_padding-horizontal);
+	border: var(--chip_border-width) solid var(--chip_outline_color);
+	border-radius: var(--chip_corner-radius);
 
-	@include darkTheme() {
-		background-color: var(--surface_primary_emphasis-low);
-		border-color: transparent;
-	}
+	@include transition(color background-color border-color box-shadow);
 
 	&:hover {
-		background-color: var(--surface_primary_emphasis-medium);
-		border-color: transparent;
-		color: var(--foreground_emphasis-high);
+		background-color: var(--chip_background-color_hovered);
+		color: var(--chip_foreground-color);
 	}
 
 	&:focus-visible {
-		background-color: var(--background_focus);
-		border-color: transparent;
-		box-shadow: 0 0 0 var(--spc-1x) var(--focus-outline_primary);
-		color: var(--foreground_emphasis-high);
+		background-color: var(--chip_background-color_focused);
+		border-color: var(--chip_focus-outline_color);
+		border-width: var(--chip_focus-outline_width);
+		color: var(--chip_foreground-color);
 	}
 
 	&:active {
-		background-color: var(--surface_primary_emphasis-high);
-		color: var(--foreground_emphasis-high);
+		background-color: var(--chip_background-color_pressed);
+		color: var(--chip_foreground-color);
+	}
+
+	&:disabled {
+		background-color: var(--chip_background-color_disabled);
+		border-color: var(--chip_outline_color_disabled);
+		color: var(--chip_foreground-color_disabled);
 	}
 }
 
-.tag:has(picture) {
-	gap: var(--spc-2x);
+.chip > svg {
+	height: var(--chip_icon_size);
+	width: var(--chip_icon_size);
 }
```

Comment:
If everything lives in `.chip_content` as declared in the component, this won't select an svg; i.e. `<Chip><svg/> Hello</Chip>` will create:
```html
<a class="chip">
    <span class="chip_content">
        <svg/> Hello
    </span>
</a>
```

I might recommend using an `icon` prop instead, i.e. `<Chip icon={<svg/>}>Hello</Chip>` to output:
```html
<a class="chip">
    <svg/>
    <span class="chip_content">
        Hello
    </span>
</a>
```